assets/app/models/Document.js

```diff
@@ -2,27 +2,68 @@ var _ = require('underscore');
 var Backbone = require('backbone');
 var yaml = require('yamljs');
 
+var fileTypes = {
+  'md': ['md', 'markdown'],
+  'yml': ['yml', 'yaml']
+};
+
+var yamlDefaults = [
```

Comment:
I don't think we want to hardcode these values (`yamlDefaults`) into this view because it can change from site to site. I think any site developer should be able to specify which layouts are available to the federalist editor by specifying in the `_config.yml` file.